src/test/java/csvDataDriven/CSVExercise.java

```diff
@@ -0,0 +1,66 @@
+package csvDataDriven;
```

Comment:
@andreea-qa we're going to have a few examples of data-driven testing, right? Let's do a pakcage called `com.data.driven` and then we can just put the examples in there accordingly? Or will that get too messy with so many examples?

Reply:
@nadvolod I was thinking to separate CSV from YML from class data, I feel it's cleaner. But we can have them all in the same place too